docs/installation.md

```diff
@@ -21,17 +21,17 @@ Weave GitOps currently supports SaaS versions of GitHub and GitLab.
 To install the `Gitops` CLI, please follow the following steps:
 
 ```console
-curl --silent --location "https://github.com/weaveworks/weave-gitops/releases/download/v0.6.2/tmp
+curl --silent --location "https://github.com/weaveworks/weave-gitops/releases/download/v0.6.2/gitops-$(uname)-$(uname -m).tar.gz" | tar xz -C /tmp
 sudo mv /tmp/gitops /usr/local/bin
 gitops version
 ```
 
 You should see:
 
 ```console
-Current Version: v0.6.1-10-g35945c8b
-GitCommit: 35945c8b
-BuildTime: 2022-01-25_22:13:54
+Current Version: v0.6.1-rc1-2-gcf7b11b8
```

Comment:
Shouldn't this be `v0.6.2` ?

Reply:
yeah sorry. I am used to this working automatically.

Reply:
Fixed it.